r/R/dictionary.R

```diff
@@ -34,6 +34,19 @@ DictionaryType <- R6Class("DictionaryType",
   public = list(
     ToString = function() {
       prettier_dictionary_type(DataType__ToString(self))
+    },
+    code = function() {
```

Comment:
Should `code` be an active binding? As in you get it with `x$code` instead of `x$code()`?

Reply:
it could as it's always called without arguments, but having `$code()` implies an action, i.e. making the code, instead of retrieving a property. 

I don't mind changing though.